### DISCUSSION TITLE
京东小程序中webview如何能显示出标题栏

### DISCUSSION THREAD

**Discussion by huanyu** [NervJS/taro#11667] [Q & A]
文档中写明webview会自动铺满整个小程序页面，但是如何才能显示出页面的带有返回按钮的导航栏呢。我的小程序只能全屏展示网页的内容，没有了导航栏只能通过手势返回上一页

**Comment by Chen-jj** [comment]
@huanyu 试试原生写法？先区分是 Taro 还是京东小程序的问题咯

**Comment by huanyu** [comment]
抱歉，是京东小程序ide显示不正确误导了，功能是正常的
